src/ssl.c

```diff
@@ -11471,14 +11471,17 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
     {
         WOLFSSL_ENTER("wolfSSL_CTX_get_options");
         WOLFSSL_MSG("wolfSSL options are set through API calls and macros");
+        if(ctx == NULL)return SSL_FAILURE;
```

Comment:
To be consistent with our existing coding style, can you please place return calls on the following line, ie:

```
if (ctx == NULL)
    return SSL_FAILURE;
```

Can you also adjust this throughout this PR?  Thanks.